### DISCUSSION TITLE
socket connectio failed

### DISCUSSION THREAD

**Discussion by satyamgit1** [socketio/socket.io#5059] [General]
any one help me

**Comment by darrachequesne** [comment]
Hi! Did you check our [troubleshooting guide](https://socket.io/docs/v4/troubleshooting-connection-issues/)?

**Reply by satyamgit1** [reply]
yes but it did not work
can you check my project over a meet?

**Reply by satyamgit1** [reply]
it is working fine on localhost but it does not working on production socket connectio failed
